### DISCUSSION TITLE
Please add a hint to the documentation of tasmotas modbus bridge; startaddress has to be given in dec, not in hex notation!

### DISCUSSION THREAD

**Discussion by HPin55** [arendst/Tasmota#20379] [General]
Hello everybody and a happy new year! Also many many thanks to all contributors who developed tasmota!

I use tasmotas modbus bridge to set certain coil registers for my epever solar inverter. The necessary command looks like this:
`ModBusSend {"deviceaddress": 3, "functioncode": 5, "startaddress": 15, "type":"bit", "count":1, "values":[1]}`
From the inverter device documentation I knew the startaddress to be hex 0x0F (which ist dec 15) and since all documentation about modbus and also tasmotas documentation about the smart meter interface, especially for the meter definition uses hex notation for the registers, it took me quite some time to figure this out.
The examples given in the modbus bridge documentation involve only very low coil register addresses such as 1 or 2, therefore there is no difference between hex an dec. But in my case this was vital.
So maybe a corresponding hint could help somebody else.

**Comment by sfromis** [comment]
As numeric values in valid JSON are **always** decimal, cannot be hex, the hint is already there. Anyway, anyone can suggest wiki-style edits to docs, pencil top right on each page.

**Reply by HPin55** [reply]
Ok, thanks for the advice. I obviously need some more experience.
